langchain/vectorstores/pgvector.py

```diff
@@ -205,6 +206,15 @@ def __from(
 
         return store
 
+    def delete_embeddings(self, ids: List[str]) -> None:
```

Comment:
Not sure that deletion of embeddings should be supported. Likely a lot of things expect the embeddings to exist. An update in place / upsert should be sufficient instead